### PR TITLE
commit 666

### PR DESCRIPTION
**Pull request by Chandrima6**
committed 666

### CODE REVIEW

**Review by carriescott — CHANGES_REQUESTED** [review]
👎Where is all the code?

**Review by mugshepherd — DISMISSED** [review]
This is terrible Chandrima.  Please add Person's name.

File: friends-and-colleagues/person666.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- Javascript
+
+
+
+## Frameworks and Technologies that I know:
+
+- Angular 6
+
+
+Email me at work!
```
Comment (carriescott) [inline friends-and-colleagues/person666.md:20]: This is not enough information. Please add your email address.

**Review by carriescott — CHANGES_REQUESTED** [review]

**Comment by mugshepherd** [comment]
@Chandrima6 @pete85 @carriescott @bayramcura please take a look

**Review by carriescott — APPROVED** [review]

**Review by bayramcura — APPROVED** [review]
Great job

**Pull request closed without merging**